npm/eslint-plugin-dev/README.md

```diff
@@ -20,23 +20,20 @@ npm install --save-dev @cypress/eslint-plugin-dev
 
 ## Usage
 
-> ⚠️ Currently does **not** support ESLint version 8+
+For Eslint 8+, use version 6.x.x
+
+For Eslint 7 and below, use version 5.x.x
```

Comment:
I'm not sure about the compatibility advice!

I doubt if you have ESLint 9 compatibility set up, which requires flat files, and I don't know that getting involved with ESLint 6 and below is still worthwhile, so that would reduce to:

> ⚠️ Currently does **not** support ESLint version 9
> For Eslint 8, use version 6.x.x
> For Eslint 7, use version 5.x.x

Reply:
I would also think that is true too. I will update the readme!